components/StyledComponents/LinkWithoutDecoration.js

```diff
@@ -0,0 +1,6 @@
+const { default: Link } = require("next/link");
+const { styled } = require("styled-components");
```

Comment:
The same goes for the second import, plus it's a default, not a named.
```suggestion
import styled from "styled-components";
```